Shared/Action/TelemetryAction.swift

```diff
@@ -82,6 +82,13 @@ class GleanActionHandler: ActionHandler {
             onCompleted: nil,
             onDisposed: nil
         ).disposed(by: self.disposeBag)
+                
+        // Get legacy telemetry ID if not in extension
+        #if LOCKWISE
+            if let uuidString = UserDefaults.standard.string(forKey: "telemetry-key-prefix-clientId"), let uuid = UUID(uuidString: uuidString) {
+                GleanMetrics.LegacyIds.clientId.set(uuid)
+            }
+        #endif
```

Comment:
I am no longer with Mozilla and was pulled off of this project midway through this work, but just wanted to make a note here in case a future dev picks this PR back up. 

Before this PR is merged, this section of code needs to be fixed. As of now, we are not entering this `if` statement when launching the Lockwise app, although we should be. We only want to get the legacy telemetry ID if we are not running the extension and we need that legacy telemetry ID for the delete ping. The `GleanMetrics` file is also not currently a part of the `CredentialProvider` extension target.

Reply:
Another solution, updating to a newer version of Glean should prevent Glean from running in a App-Extension process.

Reply:
Awesome! Should be smoother if some one needs to work on this then :)